src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+
+import re
+import sys
+
+
+if len(sys.argv) < 2:
+  print("Please provide at least one source file name as argument.")
+  quit()
+
+for file_name in sys.argv[1:]:
+
+  print("Modifying format of {file} comments in place...".format(
+      file = file_name,
```

Comment:
When passing an argument to a function by keyword, omit the spaces on either side of the equals sign.